api/auth.go

```diff
@@ -140,32 +177,53 @@ func (auth *Auth) JWTHandler(next echo.HandlerFunc) echo.HandlerFunc {
 	}
 }
 
-func createPasswordVP(customerID int, password string) auth.VerifiablePresentation {
+func createPasswordVP(customerDID string) auth.VerifiablePresentation {
+	const identifier = "t.tester@example.com"
```

Comment:
Don't we have usernames in the demo to login?

Reply:
No, only password.